### ISSUE TITLE
[Feature request] Select language to check

### ISSUE THREAD

**Issue by morgandavidson** [emacs-languagetool/flycheck-languagetool#28]
Here is my config (which works fine):

```
  (use-package flycheck-languagetool
    :hook
    (text-mode . flycheck-languagetool-setup)
    (org-mode . flycheck-languagetool-setup)
    (latex-mode . flycheck-languagetool-setup)
    (markdown-mode . flycheck-languagetool-setup)
    :init
    (setq flycheck-languagetool-url "https://api.languagetool.org")
    (setq flycheck-languagetool-server-jar nil)
    (setq flycheck-languagetool-server-port nil)
    (setq flycheck-languagetool-language "fr-FR")
    :config
    (add-to-list 'flycheck-checkers 'languagetool)
    )
```
How could I add `(setq flycheck-languagetool-language "en-US")` ?

**Comment by jcs090218** [comment]
You mean to check with multiple languages in a file? 🤔 I haven't use language-tool for a while, so I don't know if that is possible.

**Comment by morgandavidson** [comment]
I mean being able to use LanguageTool french checker in a buffer when I'm writing in French. And being able to use LanguageTool English checker in another buffer when I'm writing in English.

**Comment by jcs090218** [comment]
No, unfortunately, there is no default way to do this. You will have to set it yourself,

```elisp
(setq-local flycheck-languagetool-language "en-US")
```

in France buffer,

```elisp
(setq-local flycheck-languagetool-language "fr-FR")
```

**Comment by morgandavidson** [comment]
Thank you, `M-: (setq-local flycheck-languagetool-language "en-US")` or `M-: (setq-local flycheck-languagetool-language "fr-FR")` work fine within buffer.

Could the following functionality be added as an enhancement request?

To be able to have multiple LanguageTool checkers options when in `flycheck-select-checked` (e.g. en-US, fr-FR, etc.).

**Comment by jcs090218** [comment]
> Could the following functionality be added as an enhancement request?
> 
> To be able to have multiple LanguageTool checkers options when in `flycheck-select-checked` (e.g. en-US, fr-FR, etc.).

You mean the interface to select language before starting the checker?

**Comment by morgandavidson** [comment]
I mean, the ability to choose between FR, EN or else, here: https://paste.pics/MJ2L7

**Comment by jcs090218** [comment]
Of course, I will find some time to implement this! :D

**Comment by morgandavidson** [comment]
Awesome, thank you!
